app/src/main/res/layout/content_fragment_home.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/accent">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/layout_padding_20dp"
+        android:layout_marginRight="@dimen/layout_padding_20dp">
+
+        <ImageView
+            android:id="@+id/iv_notification"
+            android:layout_width="@dimen/layout_width_25dp"
+            android:layout_height="@dimen/layout_height_25dp"
+            android:layout_alignParentRight="true"
+            android:layout_alignParentEnd="true"
+            app:srcCompat="@drawable/ic_notifications" />
+
+    </RelativeLayout>
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/total_balance"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_18sp"
+        android:layout_marginTop="@dimen/layout_padding_10dp"
+        android:layout_marginLeft="@dimen/layout_padding_30dp"
+        android:fontFamily="@font/productsans"/>
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="@dimen/text_size_60sp"
+        android:text="@string/four_twenty_dollars"
+        android:textColor="@color/white"
+        android:layout_marginLeft="@dimen/layout_padding_30dp"
+        android:layout_marginTop="@dimen/layout_padding_10dp"
+        android:fontFamily="@font/airbnb_cereal_book"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_marginTop="@dimen/layout_padding_30dp"
+        android:layout_marginLeft="@dimen/layout_padding_30dp">
+
+        <com.google.android.material.button.MaterialButton
+            android:theme="@style/Theme.MaterialComponents"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/sendit"
```

Comment:
follow proper `id` conv, btn_send_it. Let's not repeat the same mistake over and over again

Reply:
Sorry for repeating this mistake. I'll keep that in mind in future.